### PR TITLE
fix issue 54, add a option to the Cluster command to display current active endpoint

### PR DESCRIPTION
**Pull request by qx-will**
fix issue 54, add a option to the Cluster command to display current active endpoint.
will check the current configuration to fetch the active endpoint.

### CODE REVIEW

**Comment by samedder** [comment]
Fixes #54

File: src/sfctl/commands.py [hunk 1]
```diff
@@ -231,6 +231,8 @@ def load_command_table(self, args): #pylint: disable=too-many-statements
                                'sfctl.custom_cluster#{}') as super_group:
```
Comment (samedder) [inline src/sfctl/commands.py]: Please update the readme to indicate a new command has been added.

File: src/sfctl/custom_cluster.py [hunk 2]
```diff
@@ -39,6 +39,16 @@ def select_arg_verify(endpoint, cert, key, pem, ca, aad, no_verify):
     if pem and any([cert, key]):
         raise CLIError(usage)
 
+def show():
+    """
```
Comment (samedder) [inline src/sfctl/custom_cluster.py]: Can you add the help string to the helps files, just like `sfctl cluster select` has.

**Comment by samedder** [comment]
Be sure to also merge the latest changes from master.

File: src/sfctl/custom_cluster.py [hunk 3]
```diff
@@ -39,6 +39,16 @@ def select_arg_verify(endpoint, cert, key, pem, ca, aad, no_verify):
...
+    Display the current active endpoint.
+    """
+    from sfctl.config import (client_endpoint)
+    if client_endpoint() and client_endpoint().strip():
+        print("The current active endpoint is: " + client_endpoint())
```
Comment (samedder) [inline src/sfctl/custom_cluster.py]: I think for the sake of globalisation and simplicity it's okay to return the endpoint, or empty string. Adding the text before will just make it more difficult for people to parse out the endpoint.

File: src/sfctl/custom_cluster.py (hunk 2 above)
Comment (qx-will) [reply src/sfctl/custom_cluster.py]: ok..will update the help with a new PR

File: src/sfctl/custom_cluster.py (hunk 3 above)
Comment (qx-will) [reply src/sfctl/custom_cluster.py]: good point.

File: src/sfctl/custom_cluster.py (hunk 2 above)
Comment (qx-will) [reply src/sfctl/custom_cluster.py]: might need some more information for this. which file you are suggesting to update?

**Comment by samedder** [comment]
@qx-will any update here?

**Comment by qx-will** [comment]
sorry for the late response, just back on this thread. only one issue need your further comments as below: 
"Can you add the help string to the helps files, just like sfctl cluster select has." 

need your input which file to update. thanks @samedder 

for the other parts, already fixed.

File: src/sfctl/custom_cluster.py (hunk 2 above)
Comment (samedder) [reply src/sfctl/custom_cluster.py]: You need to add a new file or update the `main.py` file in `service-fabric-cli/src/sfctl/helps/` to reflect your new command. Using a doctstring in the function signature is against our convention.

**Comment by samedder** [comment]
@qx-will I responded to your comment in the review thread.

**Comment by samedder** [comment]
@qx-will Any updates here?

**Comment by qx-will** [comment]
@samedder  any chance we could have a quick skype call on this help string thing? i assume you also work inside MS? my alias:qixwang 
thanks a lot.

**Comment by talarari** [comment]
Any update on this? really useful

**Pull request closed without merging**